src/App.css

```diff
@@ -145,3 +145,45 @@ nav {
 a.active {
   text-decoration: underline;
 }
+
+.spinner-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  max-width: 65rem;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 0;
+}
+
+.spinner {
+  position: absolute;
+  display: inline-block;
+  width: 30px;
+  height: 30px;
+  margin: 10px;
+  border: 4px solid rgba(0, 0, 0, 0.1);
+  border-left-color: #b4cddd;
+  border-radius: 50%;
+  animation: donut-spin 1.2s linear infinite;
+}
+
+@keyframes donut-spin {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.search__error h2 {
+  text-align: center;
+}
+
+.gm-style img {
+  width: 100% !important;
```

Comment:
I'm still learning CSS, but some time ago, I read that `!important` value is not recommended in most cases. In [this article](https://uxengineer.com/css-specificity-avoid-important-css/), there are some ideas maybe can help you.

Reply:
hey @alxmcr, `!important `is used here because we're overriding the Google Maps' css styles - there's no other way to tweak them otherwise.